File: CodeChef/JULY20/ADAKING.cpp
#include <iostream>
using namespace std;
typedef long long ll;

int main(){
    int t,k;
    cin >> t;
    while(t--){
        cin >> k;
        int flag = 1;
        int count = 64 - k;
        for(int i=0; i<8; i++){
            for(int j = 0; j < 8; j++){
                if(count > 0){
                    cout << "X";
                    count--;
                }
                else if (flag){
                    cout << "O";
                    flag = 0;
                }
                else{
                    cout << ".";
                }
            }
            cout << "\n";
        }
    }
}

File: CodeChef/JULY20/ADAKING.py



if __name__ == '__main__':
    for t in range(int(input())):
        k = int(input())
        count = 64 - k
        flag = True 
        for i in range(8):
            for i in range(8):
                if count > 0:
                    print('X', end = "")
                    count -= 1
                elif flag:
                    print("O", end = "")
                    flag = False 
                else:
                    print(".", end = "")
            print()
        
                
        
